src/mba_fiap/adapter/cliente_rest.clj

```diff
@@ -24,13 +23,33 @@
      :headers {"Content-Type" "application/json"}
      :body result}))
 
+(defn autenticar-cliente
+  [request]
+  (let [repository (get-in request [:app-context :repository/cliente])
+        {:keys [cpf password]} (:json-params request)
+        client-data (cliente.service/buscar-por-cpf repository cpf)
+        email (:email client-data)]
+    (if email
```

Comment:
se n tem cláusula de else, o comum é  usar 'when'

Reply:
então, nesse caso aqui of else é o status 400, eu passei o when pra ca como substituo, mas ai ele cai sempre no 400 mesmo achando o email do cara